docs/content/examples/tutorials/dovecot-solr.md

```diff
@@ -0,0 +1,65 @@
+*Dovecot full text search using solr backend*
+
+Dovecot can use several fts backends to support efficient and fast full text searching of e-mails directly from the imap server. Especially if you have one or more large mail folders this can make a huge difference, since the alternative is dovecot searching through each and every email all by itself on the spot, agian and again. The latter most times means one cannot search through a large pile off emails with clients that don't store all imap mail locally, for example mobile clients like Gmail or webmail clients.
+One of these is Apache SOLR, an fast and efficient multi-purpose search indexer.
+
+Firstly you need a working solr container, for this the official docker container will do:
+
+```
+---
+
+services:
+  solr:
+    image: solr:latest
+    environment:
+      SOLR_JAVA_MEM: "-Xms2g -Xmx2g"
+    volumes:
+      - <local folder>:/var/solr
+    restart: always
+```
+
+We'll assume dms will connect internally to solr, so either append the above docker composer snippet to your dms compose.yml or make sure both containers use the same docker network.
+The enviroment setting SOLR_JAVA_MEM is optional, but solr can be quite resource hungry so the default of 512MB can be exhausted rather quickly.
+
+Once started you need to configure a solr core for dovecot:
+```
```

Comment:
Add a newline between these two and use the `bach` syntax highlighter (write `bash` after the three backticks).